app/forms/collection_form.rb

```diff
@@ -16,7 +16,7 @@ def validate(params)
 
   # Copies the values to the model and saves and indexes
   def save
```

Comment:
Why are we doing the `||=`. Is there a case where `@model` is already set?

Reply:
Yes, it can be set on form initialization per `BaseForm`.